internal/ini/visitor.go

```diff
@@ -63,7 +63,7 @@ func (v *DefaultVisitor) VisitExpr(expr AST) error {
 
 			rhs := children[1]
 
-			if rhs.Root.Type() != TokenLit {
+			if !(rhs.Root.Type() == TokenLit || rhs.Root.Type() == TokenOp || rhs.Root.Type() == TokenSep) {
```

Comment:
Hard to follow exactly what we now support. Could you add a comment, and update the ini/doc.go